unification/chain.json

```diff
@@ -29,49 +29,146 @@
       {
         "denom": "nund"
       }
-    ]
+    ],
+    "lock_duration": {
+      "time": "1814400s"
+    }
   },
   "codebase": {
     "git_repo": "https://github.com/unification-com/mainchain",
-    "recommended_version": "v1.6.3",
+    "recommended_version": "v1.8.2",
     "compatible_versions": [
-      "v1.6.3"
+      "v1.8.2"
     ],
-    "binaries": {
-      "linux/amd64": "https://github.com/unification-com/mainchain/releases/download/v1.6.3/und_v1.6.3_linux_x86_64.tar.gz"
+    "cosmos_sdk_version": "0.46",
+    "consensus": {
+      "type": "tendermint",
+      "version": "0.34"
     },
+    "cosmwasm_enabled": false,
+    "ibc_go_version": "5.3.1",
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/unification-com/mainnet/master/latest/genesis.json"
     },
     "versions": [
       {
         "name": "v1.6.3",
-        "recommended_version": "v1.6.3",
-        "compatible_versions": [
-          "v1.6.3"
+        "tag": "v1.6.3",
+        "proposal": 6,
+        "height": 2002620,
+        "consensus": {
+          "type": "tendermint",
+          "version": "0.34.21"
+        },
+        "cosmos_sdk_version": "0.45.9",
+        "ibc_go_version": "3.1.0",
+        "ics_enabled": [
+          "ics20-1"
         ],
-        "binaries": {
-          "linux/amd64": "https://github.com/unification-com/mainchain/releases/download/v1.6.3/und_v1.6.3_linux_x86_64.tar.gz"
-        }
+        "cosmwasm_enabled": false,
+        "next_version_name": "v1.7.0"
+      },
+      {
+        "name": "v1.7.0",
+        "tag": "v1.7.0",
+        "proposal": 13,
+        "height": 4727000,
+        "consensus": {
+          "type": "tendermint",
+          "version": "0.34.26"
+        },
+        "cosmos_sdk_version": "0.45.14",
+        "ibc_go_version": "3.4.0",
+        "ics_enabled": [
+          "ics20-1"
+        ],
+        "cosmwasm_enabled": false,
+        "next_version_name": "v1.8.0"
+      },
+      {
+        "name": "v1.8.0",
+        "tag": "v1.8.0",
+        "consensus": {
+          "type": "tendermint",
+          "version": "0.34.28"
+        },
+        "cosmos_sdk_version": "0.46.13",
+        "ibc_go_version": "5.3.1",
+        "ics_enabled": [
+          "ics20-1"
+        ],
+        "cosmwasm_enabled": false,
+        "next_version_name": "v1.8.1"
+      },
+      {
+        "name": "v1.8.1",
+        "tag": "v1.8.1",
+        "consensus": {
+          "type": "tendermint",
+          "version": "0.34.29"
+        },
+        "cosmos_sdk_version": "0.46.14",
+        "ibc_go_version": "5.3.1",
+        "ics_enabled": [
+          "ics20-1"
+        ],
+        "cosmwasm_enabled": false,
+        "next_version_name": "v1.8.2"
+      },
+      {
```

Comment:
Thanks for adding all this. More info is better. But can we roll v1.8.0, v1.8.1, and v1.8.2 into a single version object please? (since they're compatible/non-conensus-breaking).
Really, just keep this v.1.8.2 object, (maybe choose a generic name like 'v1.8' or better yet use the name in the proposal), and just remove the separate v1.8.1 and v1.8.0 objects above

Reply:
Next try :)